kittensFunc.js

```diff
@@ -4,25 +4,23 @@ $(function() {
         this.path = path;
         this.votes = 0;
     };
-    Cat.prototype.vote = function() {
-        ++this.vote
-    };
+
     var tracker = [];
 
```

Comment:
Did you remove or comment your instances of Cat after you got your API working? If not, be sure to clean that up at some point for some extra practice with refactoring your code.